geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/Raster.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2014 Commonwealth Computer Research, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.locationtech.geomesa.raster.data
+
+import java.awt.image.RenderedImage
+import java.util.UUID
+
+import org.geotools.geometry.jts.ReferencedEnvelope
+import org.geotools.referencing.crs.DefaultGeographicCRS
+import org.joda.time.{DateTime, DateTimeZone}
+import org.locationtech.geomesa.accumulo.index.DecodedIndexValue
+import org.locationtech.geomesa.raster.index.RasterEntry
+import org.locationtech.geomesa.raster.util.RasterUtils
+import org.locationtech.geomesa.utils.geohash.GeohashUtils
+
+trait Raster {
+  def metadata: DecodedIndexValue
+  def resolution: Double
+  def serializedChunk: Array[Byte]
+  def chunk: RenderedImage
+
+  def id = metadata.id
+
+  lazy val time = new DateTime(metadata.date.map(new DateTime(_, DateTimeZone.UTC)).getOrElse(0L), DateTimeZone.forID("UTC"))
+
+  lazy val minimumBoundingGeoHash = GeohashUtils.getClosestAcceptableGeoHash(metadata.geom.getEnvelopeInternal)
```

Comment:
A comment here is called for, or some more documentation for the used methods in GeohashUtils. This is another major conceptual piece of the module and deserves some documentation.